src/components/PsmPanel.tsx

```diff
@@ -23,9 +23,46 @@ const anchors = [
   'USDT_grv',
 ];
 
-interface Props {}
+function Eligibility({
+  status,
+  invitation,
+  acceptedIn,
+}: ReturnType<typeof inferInvitationStatus>) {
+  switch (status) {
+    case 'nodata':
+      return <p>Loading…</p>;
+    case 'missing':
+      return (
+        <p className="rounded-lg py-5 px-6 mb-4 text-base mb-3 bg-red-100 text-red-700">
+          To govern you must first have received an invitation to the PSM
+          Charter.
+        </p>
+      );
+    case 'available':
+      return (
+        <div className="rounded-lg py-5 px-6 mb-4 text-base mb-3 bg-yellow-100 text-yellow-700">
+          To vote you will need to accept your invitation to the PSM Charter.
+          <AcceptInvitation
+            description={(invitation as any).description}
+            // TODO validate earlier that this invitation is from this contract
+            sourceContract={psmCharterInvitationSpec.instanceName}
+          />
+          And then <b>reload the page</b>.
+        </div>
+      );
+    case 'accepted':
+      return (
+        <p className="rounded-lg py-5 px-6 mb-4 text-base mb-3 bg-green-100 text-green-700">
```

Comment:
I know this is merged but potential improvement for next time is to pull these reused classnames into a class like 

```
.status-text {
  @apply rounded-lg py-5 px-6 mb-4 text-base mb-3
}
```
or at least a const and `clsx`.